docs/datascience/nb-in-the-browser.md

```diff
@@ -0,0 +1,73 @@
+
+Visual Studio Code supports working with [Jupyter Notebooks](https://jupyter-notebook.readthedocs.io/en/latest/) on the desktop, and extends to various browser-based platforms like [Codespaces](https://github.com/features/codespaces) and [VS Code for the Web](https://code.visualstudio.com/docs/editor/vscode-web).
+
+Using these browser-based platforms enables you to work on your notebooks (and beyond!) without having to install anything on your machine. You can read more about each of options and how to get started below.
+- [Jupyter notebooks on **Codespaces**](#jupyter-notebooks-on-codespaces)
+- [Remote tunneling with **VS Code Server** on VS Code for the Web](#remote-tunneling-with-vs-code-server)
+- [Connect to a remote **Jupyter server** on VS Code for the Web](#connect-to-a-remote-jupyter-server)
+
+## Jupyter notebooks on Codespaces
+
+Developing Jupyter notebooks in VS Code can be done entirely through a web-based interface using [Codespaces](https://github.com/features/codespaces), a cloud-hosted development environment that is secure and configurable with free compute resources (more on Codespaces [monthly usage quotas](https://docs.github.com/en/billing/managing-billing-for-github-codespaces/about-billing-for-github-codespaces)). This means you can take advantage of all VS Code functionalities, including Jupyter notebooks support, without installing anything on your machine. To get started quickly,
+1. Navigate to https://github.com/codespaces
+2. Under **Explore quick start templates**, select **Use this template** for **Jupyter Notebook**. If you don't see the **Jupyter Notebook** template, click **See all** and search for the template.
+    ![Use Codespaces Jupyter Notebook Template](images/nb-in-the-browser/codespaces-jupyter-template.png)
+3. That's it! A codespace will be created for you to get started!
+   > **Tip**: You can customize your project by committing [configuration files](https://docs.github.com/en/codespaces/setting-up-your-project-for-codespaces/introduction-to-dev-containers) to your repository (often known as Configuration-as-Code), which creates a repeatable codespace configuration for all users of your project. You can also see examples of predefined configurations on the [devcontainers/images repository](https://github.com/devcontainers/images/tree/main/src) to use and/or amend as needed for your project.
+
+## Jupyter notebooks on VS Code web (vscode.dev)
+
+You can also use [VS Code for the web](https://code.visualstudio.com/docs/editor/vscode-web) by going to https://vscode.dev or https://github.dev (available by typing '.' when viewing a repo on https://github.com ), which gives you options to connect to the following types of kernels for your notebook:
+1. [Remote tunneling with VS Code Server](#remote-tunneling-with-vs-code-server)
+2. [Remote tunneling with Jupyter server](#connect-to-a-remote-jupyter-server)
+
+### Remote tunneling with VS Code Server
```

Comment:
Can you connect to a server via a remote tunnel from the VS Code Desktop as well?

Reply:
Added a mention for it